include/RHF/DIISRHFSCFSolver.hpp

```diff
@@ -32,10 +32,11 @@ namespace GQCP {
 
 class DIISRHFSCFSolver : public RHFSCFSolver {
 private:
+    size_t minimum_subspace_dimension;
```

Comment:
Add the descriptions from the constructor to the member variables:
 >*  @param minimum_subspace_dimension       the minimum number of Fock matrices that have to be in the subspace before enabling DIIS
     *  @param maximum_subspace_dimension       the maximum DIIS subspace dimension before the oldest Fock matrices get discarded (one at a time)